healthcare.management.system/web-app-module/pom.xml

```diff
@@ -63,6 +63,23 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Great. This is the best place to put this part of the code, since what i does is to bundle an already build frontend application together with the Java Webapp Backend Server